backend/app/main/schemas.py

```diff
@@ -74,6 +74,8 @@ class FoodCollectCreate(FoodCollectBase):
 
 class FoodCollect(FoodCollectBase):
     id: int
+    trays: List[Tray] = []
+    foods: List[Food] = []
```

Comment:
FoodCollect shouldn't have a list of foods and trays

many foods point to 1 collect.